apps/web/src/app/core/core.module.ts

```diff
@@ -106,6 +107,10 @@ import { WebPlatformUtilsService } from "./web-platform-utils.service";
       useClass: LoginService,
       deps: [StateService],
     },
+    {
+      provide: CollectionAdminService,
+      useClass: CollectionAdminService,
+    },
```

Comment:
Can be simplified to:

```suggestion
    CollectionAdminService,
```

which is equivalent, where the DI token (injected via the constructor of components) and implementation are the same.

Or you can use `providedIn` like you did with the GroupServices, although I'm not sure what our preference is these days (I asked the question in `#code`).